client/signup/steps/page-picker/index.tsx

```diff
@@ -122,19 +120,16 @@ interface PageCellType {
 function PageCell( { pageId, popular, required, selectedPages, onClick }: PageCellType ) {
 	const translate = useTranslate();
 	const selectedIndex = selectedPages.indexOf( pageId );
-	const totalSelections = selectedPages.length;
 	const isSelected = Boolean( selectedIndex > -1 );
-	const isDisabled =
-		! config.isEnabled( 'difm/allow-extra-pages' ) && PAGE_LIMIT <= totalSelections;
 	const title = useTranslatedPageTitles()[ pageId ];
 	const description = useTranslatedPageDescriptions( pageId );
 
 	return (
-		<GridCellContainer isSelected={ isSelected } isClickDisabled={ isDisabled }>
+		<GridCellContainer isSelected={ isSelected } isClickDisabled={ false }>
```

Comment:
Minor: What do you think about removing the `isClickDisabled` prop from the `GridCellContainer` & `BrowserView` components?

Reply:
I feel like we might need that in the future :)